src/workerd/server/tests/python/hello/worker.py

```diff
@@ -0,0 +1,4 @@
+def test():
+    # This just tests that nothing raises when we run this. It isn't great though
+    # because we don't test whether we printed anything.
+    print("Does this work?")
```

Comment:
Does wd-test not give us the ability to check the response like we have with ew-test?

Reply:
Not directly. The way I've done this with other tests is to set it up so that the test itself runs in a subrequest binding, then use the test handler itself to invoke the subrequest. Look, for instance, at the abortable-fetch tests as an example.

Reply:
Thanks for the tip!

Reply:
I added some TODOs for this.